src/CoreFx.Private.TestUtilities/src/System/TheoryExtensions.cs

```diff
@@ -0,0 +1,39 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using Xunit;
+
+namespace System
+{
+    public static class TheoryExtensions
+    {
+        /// <summary>
+        /// Converts an IEnumerable<typeparamref name="T"/> into an Xunit theory compatible enumerable.
+        /// </summary>
+        public static TheoryData ToTheoryData<T>(this IEnumerable<T> data)
```

Comment:
>ToTheoryData< [](start = 33, length = 13)

is it possible we add this to the xunit extensions instead?

[https://github.com/dotnet/buildtools/tree/master/src/xunit.netcore.extensions](https://github.com/dotnet/buildtools/tree/master/src/xunit.netcore.extensions)